Packs/Armis/Integrations/ArmisEventCollector/ArmisEventCollector.py

```diff
@@ -578,6 +562,8 @@ def main():  # pragma: no cover
     fetch_start_time = handle_from_date_argument(from_date) if from_date else None
     parsed_interval = dateparser.parse(params.get('deviceFetchInterval', '24 hours')) or dateparser.parse('24 hours')
     device_fetch_interval: timedelta = (datetime.now() - parsed_interval)  # type: ignore[operator]
+    look_back = arg_to_number(params.get('eventFetchInterval')) or DEFAULT_INTERVAL
```

Comment:
Just perusing a bit here, but won't this overwrite the look_back variable from ~10 lines back?

Reply:
You're right, it was a typo.
However, we're going to implement a different mechanism so this pr is going to get closed.